src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          class="logo" 
+          src="images/logo.png" 
+          alt="logo">
+      </a>
+      <nav class="navbar">
+        <ul>
+          <li><a class="is-active" href="#">Apple</a></li>
+          <li><a href="#">Samsung</a></li>
```

Comment:
it would be great to add empty line between siblings
```suggestion
          <li><a class="is-active" href="#">Apple</a></li>
          
          <li><a href="#">Samsung</a></li>
```